Automattic/wp-calypso
# Add domain upsell to profile page

Related to https://github.com/Automattic/dotcom-forge/issues/1708

## Proposed Changes

* Add the domain upsell component to the user's profile page.
* Only add the upsell if the user has a verified email, 1 site and it's a Free site with no domain purchases. It's also not a P2 or Jetpack non-atomic site.

![domain-upsell-profile](https://user-images.githubusercontent.com/140841/222562881-090e9137-635b-4716-b93f-925dbb319628.png)


## Testing Instructions



* Confirm there are not regressions on the My Home Domain Upsell panel.
* Create a new WordPress.com account with only 1 site. Don't verify your email yet.
* Go to /me
* If you haven't verified your email, you shouldn't see the Domain Upsell on the Profile page.
* If you verify your email, you should see the upsell
* If you have an account with more or less than 1 site, you should NOT see the domain upsell on the profile page.

## Pre-merge Checklist
